src/App.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+import Page from './Page.jsx';
+
+
+function App() {
+  const [description, setDescription] = useState('');
+  const [currentId, setCurrentId] = useState(1);
+  const [todoList, setTodoList] = useState([]);
+
+  const onAdd = () => {
+    const todo = { id: currentId, description };
+    setCurrentId(currentId + 1);
+    setTodoList([...todoList, todo]);
+    setDescription('');
+  }
+
+  const onChange = (e) => {
+    setDescription(e.target.value)
+  }
+
+  const onDelete = (e) => {
+    const id = Number(e.target.dataset.id);
+    const newTodoList = todoList.filter(todo => todo.id !== id);
+    setTodoList(newTodoList);
+  }
+
+  return (
+    <Page
+      description={description}
+      todoList={todoList}
+      onAdd={onAdd}
+      onChange={onChange}
+      onDelete={onDelete} />
```

Comment:
```suggestion
      handleAdd={onAdd}
      handleChange={onChange}
      handleDelete={onDelete} />
```

위 예시와 같은 네이밍 컨벤션은 어떤가요? 실제 일을 처리하는 관점과 사용하는 입장에서 바라볼 때의 의도를 각각 드러내줄 수 있을 것 같아요.

참고:
- https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
자료 첨부 감사드립니다. 핸들러 함수의 네이밍을 할때 사용하는 입장에서 생각하도록 바꿔보겠습니다 🙏